utils/bazel/llvm-project-overlay/mlir/python/BUILD.bazel

```diff
@@ -227,6 +227,37 @@ filegroup(
     ],
 )
 
+##---------------------------------------------------------------------------##
+# EmitC dialect.
+##---------------------------------------------------------------------------##
+
+gentbl_filegroup(
+    name = "EmitCPyGen",
+    tbl_outs = [
+        (
+            [
+                "-gen-python-op-bindings",
+                "-bind-dialect=emitc",
+            ],
+            "mlir/dialects/_emitc_ops_gen.py",
+        ),
+    ],
+    tblgen = "//mlir:mlir-tblgen",
+    td_file = "mlir/dialects/EmitC.td",
+    deps = [
+        "//mlir:OpBaseTdFiles",
+        "//mlir:EmitCTdFiles",
+    ],
+)
+
+filegroup(
+    name = "EmitCPyFiles",
+    srcs = [
+        "mlir/dialects/emitc.py",
+        ":EmitCPyGen",
+    ],
+)
+
```

Comment:
you know this works or this is just your best guess? it's a good guess but FYI bazel is only generally maintained by google people. not that they wouldn't appreciate the PR but just that you don't _need_ to do this if you're not sure.

Reply:
Just a best guess, I'd rather have the Google guys add it if they need it, since I wouldn't know how to test it properly.